packages/select-react/src/Select.tsx

```diff
@@ -84,6 +85,12 @@ const noop = () => {
     return;
 };
 
+const handleMouseOver = (e: MouseEvent<HTMLButtonElement>) => {
+    // Ved mouseOver på options flytter vi fokus til dem for å unngå "dobbel fokus"
+    // der det ser ut som to forskjellige elementer er fokusert/hovered samtidig
+    (e.target as HTMLButtonElement).focus();
```

Comment:
Let the records show: vi anerkjenner at dette er hacky, men kommer ikke på noen bedre måte i farten å sørge for at vi unngår en situasjon med både fokus og hover samtidig (som begge skal visuelt være like, men helst bare én av gangen).